app/src/main/java/com/example/navadon/androidonclickevent/Option1Activity.java

```diff
@@ -44,4 +47,14 @@ private void hideKeyboardInput(View v){
         InputMethodManager imm = (InputMethodManager) getSystemService(Context.INPUT_METHOD_SERVICE);
         imm.hideSoftInputFromWindow(v.getWindowToken(), 0);
     }
+
+    public void onNextButtonClick(View view) {
```

Comment:
You may need to put this into "process(View v)" then check if that "v" equals Next or Back button by comparing their "id".